### PR TITLE
Fix mistranslations [Fix some ja translations #2857]

### PR DESCRIPTION
**Pull request by ozora-ogino**
## Description
Fixed some ja translation in `/src/intl/ja`.

## Related Issue
#2857

### CODE REVIEW

File: src/intl/ja/ja.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "about-ethereum-org": "Ethereum.orgについて",
-  "about-us": "当社情報",
+  "about-us": "私たちについて",
```
Comment (ozora-ogino) [inline src/intl/ja/ja.json:3]: 当社情報 means "about our company"
私たちについて literally means "about us"

File: src/intl/ja/page-developers-index.json [hunk 2]
```diff
@@ -23,15 +23,15 @@
   "page-developers-evm-desc": "トランザクションを処理するコンピューター",
   "page-developers-evm-link": "イーサリアム仮想マシン(EVM)",
   "page-developers-explore-documentation": "ドキュメントを探す",
-  "page-developers-feedback": "フィードバックがあれば、GitHubの問題経由またはDiscordサーバーでご連絡ください。",
-  "page-developers-frameworks-desc": "開発スピードを上げるためのツール",
+  "page-developers-feedback": "フィードバックがあれば、GitHub issueまたはDiscordでご連絡ください。",
```
Comment (ozora-ogino) [inline src/intl/ja/page-developers-index.json:26]: GitHubの問題経由 -> GitHub issue

File: src/intl/ja/page-developers-index.json [hunk 3]
```diff
@@ -23,15 +23,15 @@
...
+  "page-developers-frameworks-desc": "開発を効率化するためのツール",
   "page-developers-frameworks-link": "開発フレームワーク",
   "page-developers-fundamentals": "基本",
-  "page-developers-gas-desc": "Etherはトランザクションに電力を供給する必要がありました",
+  "page-developers-gas-desc": "トランザクションのエネルギーとして消費されるEther",
```
Comment (ozora-ogino) [inline src/intl/ja/page-developers-index.json:30]: Totally mistranslate

**Review by samajammin — APPROVED** [review]
Thank you for these!

**Comment by samajammin** [comment]
@wackerow @sorumfactory remind me, have we determined a process for pushing translations updates upstream? i.e. how do we get these updates into Crowdin?

**Comment by wackerow** [comment]
@samajammin Not that I'm aware of. @sorumfactory Should we have these double checked via Crowdin before bringing them in? These shouldn't affect other languages since English is the seed language.

**Review by wackerow — APPROVED** [review]
@ozora-ogino Thanks for your help on this! こんにちは =)

**Comment by wackerow** [comment]
@all-contributors please add @ozora-ogino for translation

**Comment by ozora-ogino** [comment]
@wackerow こんにちは！
Thank you!!

**Pull request merged**